src/agent/rustjail/src/seccomp.rs

```diff
@@ -68,7 +68,14 @@ pub fn init_seccomp(scmp: &LinuxSeccomp) -> Result<()> {
         }
 
         for name in &syscall.names {
-            let syscall_num = get_syscall_from_name(name, None)?;
+            let syscall_num = match get_syscall_from_name(name, None) {
+                Ok(num) => num,
+                Err(_) => {
+                    // If we cannot resolve the given system call, we assume it is not supported
+                    // by the kernel. Hence, we skip it without generating an error.
+                    continue;
```

Comment:
In this case, `libseccomp` returns `Could not resolve syscall name` error without the name couldn't be resolved, so if we can log it will make users easy for debugging.

Reply:
Thank you for the comment. You're right. I need to fix `libseccomp` crate.

Reply:
Might be good to log the error though?

Reply:
Thank you for the comment. Yes, I want to log the error, but the agent cannot do it without `cfd_log` file descriptor.

Reply:
In which case, I wonder if it makes sense to split  that syscall checking code out into a new `check_seccomp() -> Result<()>` function and call that before the fork. That would allow the parent to log the errors. However, that might not be a good idea if the loop takes a long time to run. If it is slow, we could always create a channel that the parent can read asynchronously. If the child finds any erorrs, it could send the details back to the parent through the channel, to allow the parent to log them.

Reply:
Thank you for the suggestion. it makes sence, but I need to make `do_init_child` and `init_child` async function.
I'll try to create it.